packages/editor/src/Combobox.tsx

```diff
@@ -20,14 +20,27 @@ const Chevron = () => (
   </svg>
 )
 
-const keys = {
+const keys: { [k: number]: string } = {
   38: 'up',
   40: 'down',
   27: 'escape',
   13: 'return',
 }
 
-export default ({
+type ComboboxOwnProps<T> = {
+  type?: string
+  name: string
+  label?: React.ReactNode
+  value: T
+  onChange: (val: string | T) => void
+  options?: T[]
+}
+type ComboboxProps<T> = ComboboxOwnProps<T> &
```

Comment:
Combobox is exported. Let's export the props too.

Reply:
Fixed in 14451fab.